docs/QA_Checklist.md

```diff
@@ -0,0 +1,59 @@
+# QA Checklist
```

Comment:
Is this a generic QA checklist that is just starting, or is this a complete QA for just payment processora/signup API?